pkg/cmd/admin/router/router.go

```diff
@@ -287,6 +287,7 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 							Spec: kapi.PodSpec{
 								ServiceAccount: cfg.ServiceAccount,
 								NodeSelector:   nodeSelector,
+								HostNetwork: true,
```

Comment:
if we do this for real, this should be an option, not hard-coded, not sure what the default should be